cli/src/main.rs

```diff
@@ -23,6 +27,13 @@ enum Commands {
 }
 
 fn main() -> anyhow::Result<()> {
+    env_logger::builder()
```

Comment:
here i am rewriting the log format, so we use `kythera-lib` logs as stdout messages, I do the same in [`refinery-cli`](https://github.com/rust-db/refinery/tree/main/refinery_cli) it's the easiest way I found back then to re-use lib logs as cli messages. In the future we can parse by log level and introduce colors if needed .